apps/demo-cra-lib/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src"]
+  "include": ["src"],
+  "compilerOptions": {}
```

Comment:
I'll remove this once https://github.com/oklas/react-app-rewire-alias/pull/43/files is merged.